gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/editor/bottomsheet/GravatarQuickEditorBottomSheet.kt

```diff
@@ -67,15 +70,17 @@ public fun GravatarQuickEditorBottomSheet(
     onAvatarSelected: () -> Unit,
     onDismiss: (dismissReason: GravatarQuickEditorDismissReason) -> Unit = {},
 ) {
-    GravatarQuickEditorBottomSheet(
-        gravatarQuickEditorParams = gravatarQuickEditorParams,
-        authenticationMethod = authenticationMethod,
-        onAvatarSelected = onAvatarSelected,
-        onDismiss = onDismiss,
-        modalBottomSheetState = rememberGravatarModalBottomSheetState(
-            avatarPickerContentLayout = gravatarQuickEditorParams.avatarPickerContentLayout,
-        ),
-    )
+    CompositionLocalProvider(LocalGravatarTheme provides mainGravatarTheme) {
```

Comment:
The only idea I had was to provide the Gravatar theme here, so if there's one provided in a parent component it will be overridden anyway. 

Are there other ways to do it?